lib/ansible/modules/network/avi/avi_cloud.py

```diff
@@ -30,47 +30,57 @@
             - The state that should be applied on the entity.
         default: present
         choices: ["absent", "present"]
+        type: str
     avi_api_update_method:
         description:
             - Default method for object update is HTTP PUT.
             - Setting to patch will override that behavior to use HTTP PATCH.
         version_added: "2.5"
         default: put
         choices: ["put", "patch"]
+        type: str
     avi_api_patch_op:
         description:
             - Patch operation to use when using avi_api_update_method as patch.
         version_added: "2.5"
         choices: ["add", "replace", "delete"]
+        type: str
     apic_configuration:
         description:
             - Apicconfiguration settings for cloud.
+        type: dict
     apic_mode:
         description:
             - Boolean flag to set apic_mode.
             - Default value when not specified in API or module is interpreted by Avi Controller as False.
         type: bool
     autoscale_polling_interval:
         description:
-            - Cloudconnector polling interval for external autoscale groups.
+            - Cloudconnector polling interval in seconds for external autoscale groups, minimum 60 seconds.
+            - Allowed values are 60-3600.
             - Field introduced in 18.2.2.
             - Default value when not specified in API or module is interpreted by Avi Controller as 60.
         version_added: "2.9"
+        type: int
     aws_configuration:
         description:
             - Awsconfiguration settings for cloud.
+        type: dict
     azure_configuration:
         description:
             - Field introduced in 17.2.1.
```

Comment:
Should have a line that actually describes the field - something like `Azure configuration settings for cloud.` This isn't a blocker for the PR, since the PR doesn't introduce the problem.